src/dark/fasta.py

```diff
@@ -2,21 +2,19 @@
 import sqlite3
 from collections.abc import Iterable, Iterator
 from hashlib import md5
-from typing import (
-    TextIO,
-    final,
```

Comment:
Why?
I don't mind, I only use it to keep pyright happy, not sure if it actually does anything